database.py

```diff
@@ -0,0 +1,219 @@
+"""
+This Module is Where all Database actions are done, which is used
+by the user.py module to get user Data.
+
+The program uses sqlite3 for a simple and great Database for storing,
+and changing data in the Person.db local file.
+
+To create a user the new_person method is used, and to check there is a user
+with the name, check_if_user_exists method is used.
+there are several get_* and change_* methods:
+
+get_days, get_primary_color, get_secondary_color, get_days_goal and 
+get_motivation.
+
+as their names say they simply selects and gets the information from the 
+database
+
+change_days, change_primary_color, change_secondary_color, change_days_goal
+and change_motivation
+
+again, as their names say they select and change the information in the database
+"""
+import sqlite3
+class DataBaser(object):
```

Comment:
Yes it could, but the name databaser is more beautiful :smile: 
The name is derived from words like driver "someone that drives or do all things needed to drive a car"
So databaser does all database things needed by the program.
But yes it can be changed for better understanding.

Reply:
uuh, I like the idea, and I always enjoy playing with words, but I think for making it easy for people to use in the future, just calling it database might be the better way to go